contrib/contrib_build_config.bzl

```diff
@@ -42,4 +42,10 @@ CONTRIB_EXTENSIONS = {
     #
 
     "envoy.matching.input_matchers.hyperscan":                  "//contrib/hyperscan/matching/input_matchers/source:config",
+
+    #
+    # Connection Balance extensions
+    #
+
+    "envoy.network.connection_balance.dlb":                                        "//contrib/dlb/source:connection_balancer",
```

Comment:
please keep the formatting/spacing consistent

Reply:
OK.